feat: add how-to-integrate-autoware (#103)
autowarefoundation/autoware-documentation: docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md

# How to integrate Autoware with your vehicle

> This page demonstrates how to intagrate Autoware with the real vehicle.

Reviewer: typo and a fix by Grammarly
```suggestion
This page demonstrates how to integrate Autoware with a real vehicle.
```

## 1. Prepare your real vehicle hardware

Prerequisites for the vehicle:

> - car-like or diff-drive vehicle

Reviewer: Probably you should capitalize each item?

https://developers.google.com/style/lists?hl=en
```suggestion
- Car-like or diff-drive vehicle
```

- onboard computer that satisfies the prerequisites (see [here](https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/source-installation/#prerequisites))
- the following devices attached
  - actuator
  - LiDAR

>   - inertial measurement unit (optional)

Reviewer: https://developers.google.com/style/procedures?hl=en#optional-steps
```suggestion
  - Optional: Inertial measurement unit
```
